nucypher/characters/chaotic.py

```diff
@@ -377,6 +411,22 @@ def __transfer(self, disbursement: int, recipient_address: str) -> str:
                                             target_address=recipient_address,
                                             sender_address=self.checksum_address)
         txhash = receipt['transactionHash']
+        if self.distribute_ether:
+            ether = self.ETHER_AIRDROP_AMOUNT
+            transaction = {'to': recipient_address,
+                           'from': self.checksum_address,
+                           'value': ether,
+                           'gasPrice': self.blockchain.client.gas_price}
+            ether_txhash = self.blockchain.client.send_transaction(transaction)
+
+            self.log.info(f"Disbursement #{self.__disbursement} OK | NU {txhash.hex()[-6:]} | ETH {ether_txhash.hex()[:6]} "
+                          f"({str(NU(disbursement, 'NuNit'))} + {self.ETHER_AIRDROP_AMOUNT} wei) -> {recipient_address}")
+
+        else:
+            self.log.info(
+                f"Disbursement #{self.__disbursement} OK | {txhash.hex()[-6:]} |"
+                f"({str(NU(disbursement, 'NuNit'))} -> {recipient_address}")
+
```

Comment:
The log messages here are somewhat confusing. This last message and the next are identical, so when `not self.distribute_ether` we print the same log message twice. Shouldn't we unify all 3 log messages here into a single message?

Reply:
This may have been a merge mistake.  I'll tidy it up.